SingularityService/src/main/java/com/hubspot/singularity/data/RequestManager.java

```diff
@@ -133,12 +133,15 @@ private String getPendingPath(String requestId, String deployId) {
   }
 
   private String getPendingPath(SingularityPendingRequest pendingRequest) {
-    String nodeName = String.format("%s%s",
-      new SingularityDeployKey(pendingRequest.getRequestId(), pendingRequest.getDeployId()),
-      pendingRequest.getPendingType().equals(PendingType.ONEOFF) ? pendingRequest.getTimestamp()  : "");
+    String nodeName = pendingQueueKey(pendingRequest);
     return ZKPaths.makePath(PENDING_PATH_ROOT, nodeName);
   }
 
+  private String pendingQueueKey(SingularityPendingRequest pendingRequest) {
+    SingularityDeployKey deployKey = new SingularityDeployKey(pendingRequest.getRequestId(), pendingRequest.getDeployId());
+    return String.format("%s%s", deployKey.toString(), pendingRequest.getTimestamp());
```

Comment:
We most likely do. This will still change the path of things that might be currently in the pending queue on startup. So we'd need to rewrite those to their new path